### DISCUSSION TITLE
Weird issue happenign across all cameras at differnt times

### DISCUSSION THREAD

**Discussion by MrCaspan** [blakeblackshear/frigate#15057] [Ask A Question]
Every so often one of the cameras will produce a video feed like below attached photos. It stops sending audio also. Its really random and always this weird triangle in the bottom right of the screen. On a color camera all the colors go weird also. Any idea what is causing this?

![image](https://github.com/user-attachments/assets/8cf809a9-c9ba-4748-a470-a67231abe622)
![image](https://github.com/user-attachments/assets/95909674-d5d5-4f24-9675-4c0e204b92ea)

**Comment by NickM-27** [comment]
See https://github.com/blakeblackshear/frigate/discussions/12724#:~:text=I%20see%20a%20strange%20diagonal%20line%20on%20my%20live%20view%2C%20but%20my%20recordings%20look%20fine.%20How%20can%20I%20fix%20it%3F

**Reply by MrCaspan** [reply]
Okay thanks, sorry never saw this FAQ... Just so you know I do not have any height or width set on any of my cameras which in the config refrence it says the default is 'use native stream resolution'

I can try forcing it for every camera to see what happens and then try the /4 option

**Reply by hawkeye217** [reply]
If you don't want to use go2rtc, users have had better luck with setting a resolution that is actually a standard aspect ratio (16/9 or 4/3). So for example, if your streams are supposedly 800x444, try 800x450 (standard 16/9 aspect).

**Reply by MrCaspan** [reply]
I need to use go2rtc so that I can use the higher resolution camera feeds on the live pages/ I have two different ratios

640x360 16:9
640x480 4:3

Both are having issues if that means anyything. I cant set the resolutions different on the cameras they are preset but i can change the resolution for detect i jsut dont want to use a different size the the native one and put mor eload on the CPU then is needed but I can do some testing to see what works ill try the 640*360 globally

**Reply by hawkeye217** [reply]
Sorry, I should have been clearer.

If you're using go2rtc, you should rarely see these visual artifacts in your stream - only when Frigate intelligently selects the jsmpeg player for your streams. But you can mitigate the artifacts by setting your `detect` width and height to the correct dimensions for each camera.

**Reply by MrCaspan** [reply]
i am using go2rtc for all my streams and still getting this

I set detect to 640x360 globaly and so far it looks normal

**Reply by hawkeye217** [reply]
> i am using go2rtc for all my streams and still getting this
> 

Yes, even with go2rtc, you still will get the artifacting occasionally when Frigate switches to the jsmpeg player and your `detect` dimensions are incorrect.

**Reply by MrCaspan** [reply]
nope it came back. ill try the x4 option

**Reply by MrCaspan** [reply]
Okay so I have checked all my sub streams to get the exact resolution using VLC All the cameras with issues are set to 640x360 but they were coming back as 640x368 so I bumped up the detect res to the next standard 16:9 resolution 854x480 (480p) and so far so good. Will let you know.

**Reply by hawkeye217** [reply]
Great. I bet something like 656x369 would work too (16:9 aspect).

**Reply by MrCaspan** [reply]
I was going to ask but man I have been asking so many questions of you two in the last week! It was hard to tell from the [FAQ](https://github.com/blakeblackshear/frigate/discussions/12724). if it has to be a common standard 16:9 size? Sorry I just looked up the next largest 16:9 "common" resolution. Does it matter if it's a larger or smaller resolution? I assume larger as you don't lose information?

I have also reached out to Reolink again to get them to explain why they dont send a proper 16:9 res stream as it screws things up! We'll see what they say!

**Reply by hawkeye217** [reply]
It depends on your expected target aspect ratio, which in your case is 16:9. Sizing up is what always seems to fix it.
